js/app.js

```diff
@@ -5,10 +5,25 @@
  * and export dependency modules to any new scripts
  * you create.
  */
-
+import Player from './classes/player.js';
+import Enemy from './classes/enemy.js';
+import './components/touch-icon.js';
+import InfoSection from './components/info-section.js';
 import Rand from "./utils/rng.js";
-import Player from "./classes/player.js";
-import Enemy from "./classes/enemy.js";
+
+// KEVIN TO TEST ICON CLASS
```

Comment:
I can see that continuing to throw test work on app.js will start to become cumbersome. Can you make two test files (keith.js) (kevin.js) and import those files into app.js, similar to how you're importing touch-icon.js. Throw your test content in there that way we can avoid constantly editing the same file. Don't worry about my code, I'll move it into my test file in my next pull request.

Reply:
got it. good call.